easybuild/easyconfigs/m/MCR/MCR-R2013b.eb

```diff
@@ -0,0 +1,18 @@
+name = 'MCR'
+version = 'R2013b'
+
+homepage = 'http://www.mathworks.com/products/compiler/mcr/'
+description = """The MATLAB Runtime is a standalone set of shared libraries
+ that enables the execution of compiled MATLAB applications
+ or components on computers that do not have MATLAB installed."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = [
+    'http://www.mathworks.com/supportfiles/MCR_Runtime/%(version)s/',
```

Comment:
Yeah, it doesn't for me either. I'll adjust the URL in my PR to be the same as other versions.